### PR TITLE
Added Flutter + Appwrite Demo & Dev Blog

### PR DESCRIPTION
**Pull request by Jonathanvanhaaften**
New to flutter, added links to demo & blog hope you enjoy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -260,6 +260,8 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
 
 * **Login Page App with React and Appwrite** -  Created an App with complete tutorial [Source Code](https://github.com/kunals131/Login-App-With-Appwrite)
 
+* **Quit Smoking App with Flutter and Appwrite** - A simple Counting app with user auth and a Blog on what I learned [Source Code](https://github.com/Jonathanvanhaaften/flutter_appwrite_1) & [Blog Post](https://dev.to/jonathanvanhaaften/what-i-learned-by-creating-a-appwrite-flutter-demo-51ce)
```
Comment (lohanidamodar) [inline README.md:263]: The blog post is not a tutorial, the post more suitable to place in the inspiration section

**Comment by Jonathanvanhaaften** [comment]
I've updated and moved the blog to Inspiration

File: README.md [hunk 2]
```diff
@@ -195,6 +195,7 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
 * [My Hacktoberfest journey with Appwrite](https://dev.to/jhrcook/my-hacktoberfest-journey-of-with-appwrite-1k52)
 * [The Best Open-Source Alternative to Firebase – Appwrite](https://instacodeblog.com/the-best-open-source-alternative-to-firebase-appwrite/)
 * [Hacktoberfest 2021 with AppWrite](https://dev.to/pretty19/hacktoberfest-2021-with-appwrite-3978)
+* [Hacktoberfest- What I learned by creating a appwrite demo app](https://dev.to/jonathanvanhaaften/what-i-learned-by-creating-a-appwrite-flutter-demo-51ce)
```
Comment (lohanidamodar) [inline README.md:198]:
```suggestion
* [Hacktoberfest- What I learned by creating a Flutter Appwrite demo app](https://dev.to/jonathanvanhaaften/what-i-learned-by-creating-a-appwrite-flutter-demo-51ce)
```

**Comment by lohanidamodar** [comment]
THANK YOU! All changes merged 🥳

Please reach out to `eldadfux` on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags!

As a way of saying thank you, we have invited you to Appwrite organization. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Pull request merged**